packages/lego-editor/src/index.tsx

```
@@ -0,0 +1,106 @@
+ * Render the content of the editor with the same styling as within the editor
+ * @param content - the content to render as HTML
+ */
+export const EditorContent = ({ content }: { content: string }) => {
+  // Round-trip the content through TipTap to ensure it's sanitized
+  const json = generateJSON(content, extensions);
+  const html = generateHTML(json, extensions);
+
+  return (
+    <div
+      className={styles.content}
+      dangerouslySetInnerHTML={{ __html: html }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)